pkg/check/kademlia/dynamic.go

```diff
@@ -102,53 +95,105 @@ func CheckDynamic(ctx context.Context, cluster *bee.Cluster, o Options) (err err
 
 		time.Sleep(5 * time.Second)
 
-		fmt.Println("Checking connectivity")
-		err = fullconnectivity.Check(ctx, cluster)
-		if err != nil {
-			fmt.Printf("Full connectivity not present\n")
-		} else {
-			fmt.Printf("Full connectivity present\n")
-		}
-
 		topologies, err := cluster.Topologies(ctx)
 		if err != nil {
 			return err
 		}
 
 		fmt.Println("Checking Kademlia")
 		if err := checkKademliaD(topologies); err != nil {
-			fmt.Printf("Kademlia check failed: %v\n", err)
+			fmt.Printf("Dynamic kademlia check failed: %v\n", err)
```

Comment:
Is this print needed as the error is retuned?

Reply:
Will it be printed by cobra? I guess yes right?

Reply:
Yes, looks like that is the case, but actually, the exact caller is not relevant, it can be anything anytime.

Reply:
:+1: